src/docs/sphinx/quickstart.rst

```diff
@@ -267,28 +267,279 @@ must be specified using either:
+
+.. code-block:: bash
+
+   $ python3 -m venv <virtual-env-name>
+
+Since Spack will need ``clingo``, activate the python vitual environment and 
+install the package to meet this dependency:
+
+.. code-block:: bash
+
+   $ <path/to/virtual-env>/bin/pip install clingo
+
+Configuring Spack
+^^^^^^^^^^^^^^^^^
+
+Next, you must tailor the Spack configuration files. We will modify the ``compilers.yaml`` and ``packages.yaml`` files in ``scripts/spack/configs/darwin/``.
+Instead of modifying them directly, you may wish to copy these files to another location outside of the Serac repo, use them as templates for the customization, 
+and use the ``--spack-config-dir`` option to use them when invoking uberenv as described above.
+
+Example ``compilers.yaml``:
+
+.. code-block:: yaml
+
+   compilers:
+   - compiler:
+      environment: {}
+      extra_rpaths: []
+      flags: {}
+      modules: []
+      operating_system: bigsur
+      paths:
+         cc: /opt/local/bin/clang
+         cxx: /opt/local/bin/clang++
+         f77: /opt/local/bin/gfortran
+         fc: /opt/local/bin/gfortran
+      spec: clang@12.0.1
+      target: x86_64
+
+NOTES: 
+
+* The ``operating_system`` field should be set according to your macOS version. (For example, ``"mojave", "catalina", "bigsur", "monterey", "ventura"``).
+* By default, MacPorts installs packages in ``/opt/local``; the above ``paths`` need to be adjusted if you choose a different location.
+  This of course applies to the packages in ``packages.yaml`` as well.
+* As noted above, the ``port select ...`` commands will set which version of clang gets invoked by the executables ``/opt/local/bin/clang``, etc.
+  The paths above are thus valid only if you activated the ``clang`` package that matches the compiler spec.
+  Alternatively, you could set the full name and path of the executables of the desired compilers if you don't want the operation of 
+  Spack to be influenced by your MacPorts settings.
+* You should set ``spec`` to the actual version of the compiler you installed.
+* The ``target`` entry should be set to ``x86_64`` or ``m1`` depending on which architecture your machine uses.  
+
+Here is an example of ``packages.yaml``:
+
+.. code-block:: yaml
+
+   packages:
+   all:
+      compiler: [clang, gcc]
+      providers:
+         blas: [netlib-lapack]
+         lapack: [netlib-lapack]
+         mpi: [openmpi]
+
+   mpi:
+      buildable: false
+   openmpi:
+      externals:
+      - spec: openmpi@4.1.4
+         prefix: /opt/local
+   
+   netlib-lapack:
+      buildable: false
+      externals:
+      - spec: netlib-lapack@3.10.1
+        prefix: /opt/local
+   autoconf:
+      buildable: false
+      externals:
+      - spec: autoconf@2.71
+         prefix: /opt/local
+   automake:
+      buildable: false
+      externals:
+      - spec: automake@1.16.5
+         prefix: /opt/local
```

Comment:
i noticed more of the `prefix` sections throughout `packages.yaml` needs an extra space removed (so that it aligns with spec). after that i think everything else lgtm

Reply:
Oo, that was a nasty one. I think there were actual TABS in there instead of spaces. Should be good now. Nice catch!